la_funding_analysis/pipeline/plotters.py

```diff
@@ -0,0 +1,363 @@
+# File: utils/plotters.py
+"""Functions to assist with plotting.
```

Comment:
took a look at your graphs and they seemed really insightful - I wonder if its worth chatting with Luca about design standards around accessibility and using Nesta's branding i.e. Nesta's fonts etc.

Reply:
Thanks! It's already arranged that these are going to be sent to someone external and they'll take charge of design/branding but this is a great point generally